src/utils/utils.js

```diff
@@ -898,6 +903,26 @@ export function getInSelection(document, selector) {
     );
 }
 
+/**
+ * Returns an array of url Infos for url matched in the given string.
+ *
+ * @param {String} string
+ * @returns {Array}
+ */
+export function getUrlsInfosIn(string) {
```

Comment:
`getUrlsInfosIn` ?

Reply:
in the given string.

Reply:
Oups, I wanted to suggest: `getUrlsInfos`. I never saw a preposition at the end of the function name in Odoo.